ignition-rendering1.rb

```diff
@@ -0,0 +1,51 @@
+class IgnitionRendering1 < Formula
+  desc "Rendering library for robotics applications using gz11 branches"
+  homepage "https://bitbucket.org/ignitionrobotics/ign-rendering"
+  url "https://bitbucket.org/ignitionrobotics/ign-rendering/get/70c0ba0fe42a.tar.gz"
```

Comment:
this commit points to the default branch, which is still ignition-rendering0; I think you need a commit from the gz11 branch?

Reply:
Fixed, thanks!